lib/fitfree.gi

```diff
@@ -1398,6 +1398,31 @@ local l;
   fi;
 end);
 
+
+#############################################################################
+##
+#M  SylowSubgroupOp( <G>, <pi> )
+##
+## Fitting free approach
+##
+InstallMethod( SylowSubgroupOp, "fitting free",true,
+    [ IsGroup and CanComputeFittingFree,IsPosInt ],
+    -1, # rank lower than anything that would imply CanComputeFittingFree
```

Comment:
Why -1 here? And why -5 for the Normalizer method? Are you trying to rank below specific method? Or just undoing the upranking caused by CanComputeFittingFree? In my GAP install, `RankFilter(IsGroup)` is 35 and `RankFilter(CanComputeFittingFree)` equals `RankFilter(IsGroup and CanComputeFittingFree)` equals 52.

Reply:
-1 to guarantee that it will be lower than the permutation group method (probably is anyhow)

With `Normalizer`, it turns out that the pc group method can dispatch to the generic case. I had thought initially it would just be a ranking issue (thus the -5), but in the end this made it necessary to explicitly exclude the pc group case.

Reply:
OK, but then the comment should reflect that, otherwise when we run into problems with this in the future, it will be hard to  understand why this downranking happens.

Though it is indeed unnecessary right now:
```
gap> RankFilter(CanComputeFittingFree);
52
gap> RankFilter(IsPermGroup);
65
```
But of course strictly speaking it also isn't sufficient. as this ranking order could in principle reverse. That said, this seems highly unlikely, as it doesn't seem plausible that `CanComputeFittingFree` shouldn't obtain implications (there isn't really anything you can deduce from this filter, is there?).

So perhaps just drop that line?

Reply:
> Though it is indeed unnecessary right now:

> But of course strictly speaking it also isn't sufficient. as this ranking order could in principle reverse. That said, this seems highly unlikely, as it doesn't seem plausible that `CanComputeFittingFree` shouldn't obtain implications (there isn't really anything you can deduce from this filter, is there?).
> 
> So perhaps just drop that line?

In the long run.

`Centralizer` (as well as `Normalizer` below) are prominent operations, and I want to make sure this ``cross representation`` installation does not suddenly break things by overriding existing methods due to surprising implications.
I have not tried every representation of groups that currently exist.

Ranking lower seems safer.